lib/fs.js

```diff
@@ -1452,7 +1457,13 @@ function lutimesSync(path, atime, mtime) {
 
 function writeAll(fd, isUserFd, buffer, offset, length, signal, callback) {
   if (signal?.aborted) {
-    callback(lazyDOMException('The operation was aborted', 'AbortError'));
+    if (isUserFd) {
+      callback(lazyDOMException('The operation was aborted', 'AbortError'));
+    } else {
+      fs.close(fd, function() {
+        callback(lazyDOMException('The operation was aborted', 'AbortError'));
+      });
```

Comment:
Is it safe to ignore the error thrown by `fs.close` here?

Reply:
While this is a bit different, the same happens a few lines lower where if you get a write error, the close error gets swallowed and the write error gets propagated.

Reply:
however, I assume that we should make the behaviour consistent between this and the other PR

Reply:
For the sync version, the error is not swallowed. I think it never should be, I've open https://github.com/nodejs/node/pull/37460 to address this.